app.py

```diff
@@ -1,35 +1,59 @@
-from typing import List, Dict, Optional, Tuple
+from typing import List, Dict
 
-from fastapi import FastAPI
+from fastapi import FastAPI, responses
 from pydantic import BaseModel
+from submodules.model.business_objects import general
 import util
 
 app = FastAPI()
 
 
-class Request(BaseModel):
-    current_version: str
-    target_version: str
+@app.post("/update_to_newest")
+def update_to_newest() -> int:
```

Comment:
From the other api endpoints, I conclude that as return type the type of the actual content of the return value is stated and that the status_code is ignored. In that case, the return value should be string.
```suggestion
def update_to_newest() -> str:
```

Reply:
changed